### ISSUE TITLE
applications: desktop_actions not working anymore

### ISSUE THREAD

**Issue by abenz1267** [Kirottu/anyrun#65]
Hi,

i realized i havent updated anyrun in a long time... looks like i shouldn't have :-P. `desktop_actions: true` doesn't seem to work anymore.

Regards

**Comment by Kirottu** [comment]
The configuration file for the Applications file has changed since, which would make the loading of the config file fail. Do check the README of the Applications plugin to find the new schema.

**Comment by abenz1267** [comment]
Ah, apparently i NEED the max_entries option in the config. The terminal one can be omitted. This is a bit unexpected behaviour.

**Comment by Kirottu** [comment]
Yes, as the terminal can be automatically determined so it is an `Option` in the source code.
